### DISCUSSION TITLE
Fabric : How to update color of fabric.js control cursor ?

### DISCUSSION THREAD

**Discussion by niketaOptimumnbrew** [fabricjs/fabric.js#10013] [Q&A]
Hi,

I have default control cursor with white color in window , want to update cursor color from white to black.
Is it possible in fabric.js?

fabric version : 5.3.0

**Comment by asturur** [comment]
If you refer to mouse cursor, you can create a custom one that is supported by css and use that
https://developer.mozilla.org/en-US/docs/Web/CSS/cursor
